Shared/Stores/PeopleStore.swift

```diff
@@ -89,16 +92,20 @@ class PeopleStore {
     private func getAndStoreAllContactsWithUnclaimedReservations() async {
         let reservations = await Reservation.getAllUnclaimed()
         reservations.forEach { reservation in
+            if let reservationId = reservation.objectId {
+                self.reservationDictionary[reservationId] = reservation
+            }
```

Comment:
Saving the reservations so that we can find people in conversations that aren't users yet.